packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/virtualmachine/dedicatedCloud-datacenter-virtualmachine.controller.js

```diff
@@ -0,0 +1,60 @@
+import pick from 'lodash/pick';
+
+import { DEDICATED_CLOUD_DATACENTER } from '../dedicatedCloud-datacenter.constants';
+
+export default class {
+  /* @ngInject */
+  constructor($q, DedicatedCloud, $translate) {
+    this.$q = $q;
+    this.DedicatedCloud = DedicatedCloud;
+    this.$translate = $translate;
+
+    this.DEDICATED_CLOUD_DATACENTER = DEDICATED_CLOUD_DATACENTER;
```

Comment:
Constant seems not used
```suggestion
    this.$translate = $translate;
```